Week3/homework/step2-6.js

```diff
@@ -4,13 +4,11 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function flattenArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  return arr.reduce((acc, curr) => [...acc, ...curr]);
 }
 
 function flattenArray3d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  return [].concat(...arr).reduce((acc, curr) => [...acc, ...curr]);
```

Comment:
👍 But, if you like to take a step further, try to search on google on how you can implement it using `recursion`.